python/modules/IcePy/Types.cpp

```diff
@@ -1222,10 +1222,12 @@ IcePy::StructInfo::marshal(
     {
         if (!_nullMarshalValue.get())
         {
-            PyObjectHandle args{PyTuple_New(0)};
+            PyObjectHandle emptyArgs{PyTuple_New(0)};
             PyTypeObject* type = reinterpret_cast<PyTypeObject*>(pythonType);
-            _nullMarshalValue = type->tp_new(type, args.get(), 0);
-            type->tp_init(_nullMarshalValue.get(), args.get(), 0); // Initialize the struct members
+            // Create a new instance of the type
+            _nullMarshalValue = type->tp_new(type, emptyArgs.get(), nullptr);
```

Comment:
Don't we need a different set of args for `tp_init` here?

Reply:
no, we are just passing a reference to an empty list of args.